akka-docs-new/src/main/paradox/guide/actors-intro.md

```diff
@@ -0,0 +1,259 @@
+# What problems does the actor model solve? 
+
+Akka uses the actor model to overcome the limitations of traditional object-oriented programming models and meet the 
+unique challenges of highly distributed systems. To fully understand why the actor model is necessary, it helps to 
+identify mismatches between traditional approaches to programming and the realities of concurrent and distributed 
+computing. 
+
```

Comment:
Since we start with a `#` and then we do `###` these should be separated by a 

```
## Where old models fall short

Before we explain the actor model in-depth, let's take a step back and identify the problems
with common and widely-known paradigms. Once we identify the shortcomings of these old models,
we'll focus on explaining the actor model in section [....]
```

?